### PR TITLE
My first branch

### PR DESCRIPTION
**Pull request by DannieMo**
Milestone 1: Portfolio Website

Brief Description
This is the first milestone in building my portfolio website. This is a practice project at Microverse: a remote software programming school. Check my README file for more info.  😎

### CODE REVIEW

**Comment by bellom** [comment]
## Hi @DannieMo  👏  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has been marked as invalid because of the following reasons;

1. PR description is not detailed, kindly list out what you did in this project.
Example; 
image


2. README file is not detailed enough.
If check the [Microverse template](https://github.com/microverseinc/readme-template) you will see there is a section where you're supposed to add `Set up`.
So you need to add this project set up so that any non-technical user can set up your app on a local machine.
And also, add a screenshot of your web app.

Example: 
- git clone `your repo`
- cd into the folder
- open `index.html` with live server

3. To make your commit messages look more professional, you should always capitalize them.

4. Kindly add `node_modules` to `.gitignore` file, this would help to perform inline code review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,57 @@
-# Portfolio-Website
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio-Website
+
+> This is a practice project at Microverse: a remote software programming school. This project is my personal portfolio which covers my work, skills, and history. You can seek me out if you are interested in the things i do or just chatting. One paragraph statement about the project.
+
+![screenshot]![image](https://user-images.githubusercontent.com/53879944/124951902-adcbaa80-e00b-11eb-9232-6eada9006e75.png)
+(./app_screenshot.png)
```
Comment (arslanbisharat) [inline README.md]: - [ ] This link is broken. Please work on it.

File: README.md [hunk 2]
```diff
@@ -1 +1,61 @@
...
+## Built With
+
+- HTML
+- CSS 
+* CSS Flexible Box Layout
+
+ ## Live Demo 
+ To get a local copy up and runnig follow the following steps:
+
+[Live Demo Link](https://livedemo.com)
```
Comment (arslanbisharat) [inline README.md]: - [  ] It would be great if you can add live link of the project.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/e744810f7a.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+   <header>
+       <p>Daniel Owiri</p>
```
Comment (arslanbisharat) [inline index.html]: - [ ] The logo should be a link enclosed in `a tag`

File: index.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (arslanbisharat) [inline index.html]: - [ ] [Optional] It would be great if you add a proper title of the prokect here.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please add node_modules in .gitignore to clean your project. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @DannieMo , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 # Required Changes ♻️

- [ ] Please make sure `node_modules` folder is properly ignored from the git track before making a git push and requesting a re-review
_/node_modules_

### Optional suggestions
I suggest you deploy your project with Github pages

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by carlos-ssh — APPROVED** [review]
Hi @DannieMo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**